src/dapps/Salad/Salad.scss

```diff
@@ -0,0 +1,44 @@
+@import '~@/scss/GlobalVariables';
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400&display=swap');
+%shared-text {
+    font-family: "Avenir Next";
+    font-style: normal;
+    font-weight: normal;
+}
+
+%shared-links {
+    a {
+        color: #7073b5 !important;
+    }
+}
+
+#salad-dapp-container {
+    border: 2px solid #7073b5;
+    background-color: #d0ece9;
+    box-sizing: border-box;
+    width: 715px;
+    height: auto;
+    left: 0px;
+    color: #28464F;
+    @extend %shared-text;
```

Comment:
Nice!
Just opened another [little PR](https://github.com/enigmampc/MyEtherWallet/pull/7) for more of this in a global file.

Then we can standardize on primary/secondary buttons, errors etc in there too.

Reply:
yeah, sounds great.